examples/pytorch/ggnn/README.md

```diff
@@ -0,0 +1,62 @@
+# Gated Graph Neural Network (GGNN)
+
+- Paper link: https://arxiv.org/pdf/1511.05493.pdf
+
+## Dependencies
+- PyTorch 1.0+
+
+- DGL 0.3.1+
+
+## Solving bAbI tasks
+
+- bAbI dataset is generated randomly by [script](https://github.com/facebook/bAbI-tasks). For convenience, we have generated the dataset we used in this example [here](https://s3.us-east-2.amazonaws.com/dgl.ai/models/ggnn_babi_data.zip). Unzip the file to current directory.
+
+- For task 4, there are four question types: n, s, w, e, their corresponding question id are ranged from 0 to 3. For each question type, run the following commands.
```

Comment:
Elaborate a bit about the semantic meanings of the question types. I think this sentence will be nice to quote

> Some tasks have multiple question types, for example Task 4 which has 4 question types: e, s, w, n. For such tasks we simply train a separate GGNN for each task.

*Grammar correction*: "are ranged from" should be "ranges from".